src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx

```diff
@@ -321,8 +326,7 @@ class GlobalSelectionHeader extends React.Component {
       <Header className={className}>
         <HeaderItemPosition>
           {forceProject && this.getBackButton()}
```

Comment:
can this not exist directly under `Header` now? That way it won't affect the layout of the first `HeaderItem` and cause it to be different.

Reply:
Yeah, would you mind doing this in a follow up?